lighthouse-cli/test/fixtures/perf/delayed-element.js

```diff
@@ -33,6 +33,7 @@ setTimeout(() => {
   const top = document.getElementById('late-content');
   top.appendChild(imgEl);
   top.appendChild(textEl);
+  top.setAttribute('height', 18); // height can be flaky so we set it manually
```

Comment:
Setting the height changed the layout shift results so I set the height here.  Unfortunately this has not fixed the flaky smoke test.

@patrickhulce did you have another way of setting the height in mind?

Reply:
`height` is only a special attribute for specific elements, you should use `.style.height = '18px'` for a `<div>`

but that's also not the element we want to set a fixed height for, isn't it? I thought the `span` *inside* `top` is the one that's the one with a flaky height?

Reply:
Ah yes your right.  I must have misread the error.

Reply:
```
{
          "traceEventType": "layout-shift",
          "devtoolsNodePath": "0,HTML,1,BODY,1,DIV,1,SPAN",
          "selector": "body > div#late-content > span",
          "nodeLabel": "Sorry!",
          "snippet": "<span style=\"height: 18px;\">",
          "boundingRect": {
            "top": 426,
            "bottom": 443,
            "left": 8,
            "right": 49,
            "width": 41,
            "height": 17
          },
          "score": 0.003006266276041667,
          "nodeId": 13
}
```
The height is clearly set to 18px in the snippet but the `boundingRect.height` shows 17. Am I missing something?

Reply:
`height` doesn't apply to `display: inline` elements. It's a `<span>` so the default display is `inline`.

You'll need to switch it to a `<div>` or apply `display: inline-block` or `display: block`

Reply:
Looks like smoke tests are passing now.  Big thanks!